src/layouts/Account-HEDA Organization Layout.layout

```diff
@@ -27,6 +27,10 @@
                 <behavior>Edit</behavior>
                 <field>ParentId</field>
             </layoutItems>
+            <layoutItems>
```

Comment:
@kdy618  @lifewithryan  This layout is used by all Account record types except for Household right now - are we sure we actually want to add this field to the page layout? (I'm not entirely sure what this field is used for, to be honest....)

Reply:
@ceiroa So it's okay for this field to be on all Account page layouts?

Reply:
Since the PM originally said so, and since I don't see any reason not to, I'd say yes. It just points back to the first Contact that was made a child of the Account.